python-semantic-release/python-semantic-release
# chore(config): deprecate legacy tag parser w/ configuration warning

## Purpose


- Provide warning to user about the future removal of the tag parser 
- Bring down test execution time by eliminating all tag parser test code (as it is no longer supported, no need to test for it)

## Rationale


The tag parser has been around since the beginning of PSR and ultimately the implemetation has been enhanced in the form of the emoji parser.  Ultimately each parser type adds significant testing time through parameterization and repo creation.  To limit our growing test execution time, I would like to deprecate and eventually remove the tag parser to lower testing time and additional maintainability requirements.  

## How did you test?


Manually ensured the log message is displayed to the user

## How to Verify


1. change your `semantic_release.commit_parser` setting to `tag`
2. run the changelog (could be any command) in noop and see the warning message

```sh
semantic-release --noop changelog
```
